_config.yml

```diff
@@ -185,7 +185,7 @@ scheme: Muse
 
 # ---------------------------------------------------------------
 # Sidebar Settings
-# See: https://theme-next.org/docs/theme-settings/sidebar/
+# See: https://theme-next.org/docs/theme-settings/sidebar
```

Comment:
Wait a minute... this url is for users, not search engine. If users copy & past `https://theme-next.org/docs/theme-settings/sidebar` in their browser, the browser will request `https://theme-next.org/docs/theme-settings/sidebar`, and then receive a 301 rewrite to `https://theme-next.org/docs/theme-settings/sidebar/`, and the browser will request again. Which means: delete the `/` at the end of the url will cause an extra http request.

Reply:
What are u talking about?
https://check-host.net/check-http?host=https://theme-next.org/docs/advanced-settings